module-gui/gui/core/FontManager.cpp

```diff
@@ -152,11 +153,13 @@ namespace gui
     {
         loadFonts(baseDirectory);
 
-        auto fallback_font = find(getFontName(defaultFontName));
-        if (fallback_font != nullptr) {
-            for (auto font : fonts) {
-                font->setFallbackFont(fallback_font);
-            }
+        auto fallback_font = find(fallbackFontName);
+        if (not fallback_font) {
+            return false;
+        }
+
+        for (auto font : fonts) {
```

Comment:
```suggestion
        for (auto& font : fonts) {
```

Reply:
Without it, each font will be created and then destroyed instead of just passing reference.